meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio/stm32mp15-disco/lmp.cfg

```diff
@@ -4,7 +4,9 @@ CONFIG_USE_BOOTCOMMAND=y
 CONFIG_BOOTDELAY=-2
 CONFIG_FIT=y
 CONFIG_FIT_SIGNATURE=y
+CONFIG_FIT_SIGNATURE_STRICT=y
 CONFIG_FIT_VERBOSE=y
+CONFIG_RSA=y
```

Comment:
is it needed for kernel FIT image?

Reply:
Yup, not enabled by default on this u-boot rev.